extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/BodyHandlerConfig.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.vertx.http.runtime;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class BodyHandlerConfig {
+
+    /**
+     * If `true` the files sent using `multipart/form-data` will be stored locally under `uploadsDirectory` and will be
+     * made available via `io.vertx.ext.web.RoutingContext.fileUploads()`
```

Comment:
It would be nice if you could have a short sentence first (ending with a dot) then the details.

It will be better for automatic doc generation.

Reply:
@gsmet would you please propose a solution? I am a bit unsure about your definition of short and about what you see as a good split into a 'short sentence' and 'details'. I'd be ready to add a period at the end to make the parser happy, but that's probably not what you'd like.

Reply:
Something like:
```
Whether the files sent using {@code multipart/form-data} will be stored locally.
<p>
If true, they will be stored in {@code quarkus..<insert the right property here>} and will be made available via {@code io.vertx.ext.web.RoutingContext.fileUploads()}.
```
This should be done consistently for all the doc here.

BTW, be careful: either you use plain old Javadoc or Asciidoc but if you use Asciidoc in the doc, you need to add the `@asciidoclet` tag.

Reply:
Thanks, polished the JavaDoc in 0309b80 following your advice.